stylesheets/components/Avatars/avatars.css

```diff
@@ -0,0 +1,26 @@
+/* Avatars */
+.avatar {
+    border-radius: 50%;
+    position: relative;
+    border: 5px solid var(--material-color-grey-50);
+  }
+  
+  .avatar .--small {
```

Comment:
Are you using class `.--small`, `--medium` and `.--large` anywhere outside `avatars.css` file?

if not it would be better to name this classes as avatar-small, and so on

Reply:
Yeah, it would be nice if keep the class name related to the avatar only.
Rest everything looks good.

Reply:
> Are you using class `.--small`, `--medium` and `.--large` anywhere outside `avatars.css` file?
> 
> if not it would be better to name this classes as avatar-small, and so on

No Tahir. Have rectified the class names for the same.

Reply:
> Yeah, it would be nice if keep the class name related to the avatar only. Rest everything looks good.

Thank You Tanishq. The changes have been made. Appreciate the feedback.